website/documentation/contribute/code/roles/assets/_issue_template.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Membership Request
+about: Request a membership status in the project
```

Comment:
I'm wondering if we should add this issue template only once in a central repository (maybe this one) instead of copying it to individual repositories – and then open the membership requests in this central repository.
We could add a field to the template for specifying the relevant projects. This would prevent outdated templates and reduce effort when adjusting the process.

Reply:
Good suggestion! `gardener/documentation` might not be the perfect place for such requests but we can create a new repo `gardener/org` which we can use for further purposes (see also [this comment](https://github.com/gardener/documentation/pull/715#discussion_r2325113495)).